test/hardhat/helpers/crypto/abe-bn254.ts

```diff
@@ -1,12 +1,11 @@
 /**
  * Conditional Encryption / Attribute-Based Encryption for marshmallow
  */
-
-import { encrypt_towards_identity_g1, G1, G2, get_identity_g1, serializeCiphertext } from "./ibe-bn254"
+import { encrypt_towards_identity_g1, G1, G2, get_identity_g1, serializeCiphertext } from "./ibe-bn254";
```

Comment:
have we a linter enforcing `;`?

Reply:
We have prettier setup in `package.json`

```json
"lint:fix": "npm run lint:forge:fix && npm run lint:hardhat:fix",
"lint:forge:fix": "forge fmt",
"lint:hardhat:fix": "prettier --write test/hardhat"
```